Makefile

```
@@ -45,20 +51,27 @@ clean: ## Stop and remove containers and images
 logs: ## View container logs
 	docker logs -f $(CONTAINER_NAME)
 
-prod-deploy: ## TODO: Deploy to GCP Cloud Run
-	@echo "TODO: Implement GCP Cloud Run deployment"
+deploy: ## Deploy to GCP Cloud Run (publicly accessible)
```

checkmake (0.2.2)

[warning] Target body for "deploy" exceeds allowed length of 5 (23).

(maxbodylength)